content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -500,6 +500,22 @@ There should be 1 rule in `KUBE-SERVICES`, 1 or 2 rules per endpoint in
 per endpoint, and a few rules in each `KUBE-SEP-(hash)` chain.  The exact rules
 will vary based on your exact config (including node-ports and load-balancers).
 
+#### Ipvs
+
+```shell
+u@node$ ipvsadm -ln
+Prot LocalAddress:Port Scheduler Flags
+  -> RemoteAddress:Port           Forward Weight ActiveConn InActConn
+...
+TCP  10.0.0.92:80 rr
+  -> 172.17.0.2:937               Masq    1      0          0
+  -> 172.17.0.4:937               Masq    1      0          0
+  -> 172.17.0.5:937               Masq    1      0          0
+...
+```
+
+There should be rules with "`clusterIP:Port` -> `podIP:Port`".
```

Comment:
Please explain it a bit more.

IPVS proxy will create a virtual server for each service address(e.g. Cluster IP, external IP, NodePort IP, Load Balancer IP etc.) and some corresponding real servers for endpoints of the service, if any.